src/session_openssl.c

```diff
@@ -43,6 +44,36 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+void
+tls_keylog_write_line(const SSL *ssl, const char *line)
+{
+    (void)ssl;
```

Comment:
We use `UNUSED()` for the parameter instead.

Reply:
ok done